tools/policy_table_validator/main.cpp

```diff
@@ -59,12 +59,11 @@ int main(int argc, char** argv) {
     exit(READ_ERROR);
   }
 
-  Json::Reader reader;
+  utils::JsonReader  reader;
   Json::Value value;
+  bool parce_result = reader.parse(json_string, &value);
```

Comment:
i know it was there before but while we are changing this lets update `parce` -> `parse`

Reply:
done in [2d489f0](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/2d489f052063a2b121cd3a78ba26e2bad2fccc29)